### DISCUSSION TITLE
Lodestar Standup 2024-41 (October 22, 2024)

### DISCUSSION THREAD

**Discussion by philknows** [ChainSafe/lodestar#7166] [Standups]
### Meeting Info
[October 22, 2024, 14:00-15:00 UTC](https://savvytime.com/converter/utc-to-germany-berlin-united-kingdom-london-ny-new-york-city-ca-san-francisco-thailand-bangkok-japan-tokyo-australia-sydney/oct-22-2024/2pm)
Zoom: shared on #lodestar-general ChainSafe Discord channel at 14:00 UTC

### Agenda
- v1.23.0-rc.1 status
  - #7164 
  - #7134 
  - #7181 
  - #6483 

- Open Discussion
  - Data from Stakely 
  - https://github.com/ChainSafe/lodestar/pull/7165
- Team Updates
- Closing Remarks

### Instructions
Please post here any topics to be added for Standup discussions. Examples of agenda items are issues, discussions, proposals which require feedback, consensus from the team and/or community members.

**Comment by amlevy327** [comment]
Discuss data from Stakely

**Comment by philknows** [comment]
Closing for #7219
